axelor-base/src/main/java/com/axelor/apps/base/service/user/UserServiceImpl.java

```diff
@@ -39,6 +54,10 @@
 
   public static String DEFAULT_LOCALE = "en";
 
+  private static AppSettings appSettings = AppSettings.get();
+  private static Pattern pwdPattern =
+      Pattern.compile(MoreObjects.firstNonNull(appSettings.get("user.password.pattern"), ""));
```

Comment:
Please add a default regex for respect GDPR (https://www.cnil.fr/fr/authentification-par-mot-de-passe-les-mesures-de-securite-elementaires)